libraries/datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Constants.java

```diff
@@ -128,6 +145,78 @@ public class SW360Constants {
     public static final Boolean MAIL_REQUEST_FOR_PROJECT_REPORT;
     public static final Boolean MAIL_REQUEST_FOR_COMPONENT_REPORT;
 
+    public static final Boolean MAINLINE_STATE_ENABLED_FOR_USER;
+    public static final Boolean IS_BULK_RELEASE_DELETING_ENABLED;
+    public static final Boolean IS_FORCE_UPDATE_ENABLED;
+    public static final boolean IS_COMPONENT_VISIBILITY_RESTRICTION_ENABLED;
+    public static final boolean IS_ADMIN_PRIVATE_ACCESS_ENABLED;
+    public static final Boolean DISABLE_CLEARING_FOSSOLOGY_REPORT_DOWNLOAD;
+    public static final String FRIENDLY_RELEASE_URL;
+
+    // Authorization server
+    private static final String DEFAULT_WRITE_ACCESS_USERGROUP = UserGroup.SW360_ADMIN.name();
+    private static final String DEFAULT_ADMIN_ACCESS_USERGROUP = UserGroup.SW360_ADMIN.name();
+    public static final String SW360_LIFERAY_COMPANY_ID;
+    public static final String CONFIG_ACCESS_TOKEN_VALIDITY_SECONDS;
+    public static final UserGroup CONFIG_WRITE_ACCESS_USERGROUP;
+    public static final UserGroup CONFIG_ADMIN_ACCESS_USERGROUP;
+
+    // CLI Constants
+    public static final String CLI_RELEASE_EXTERNAL_ID_CORRELATION_KEY;
+
+    // CVE Constants
+    public static final int CVE_VENDOR_THRESHOLD;
+    public static final int CVE_PRODUCT_THRESHOLD;
+    public static final int CVE_CUTOFF;
+
+    // REST Constants
+    public static final String REST_API_TOKEN_HASH_SALT;
+    public static final String REST_API_TOKEN_MAX_VALIDITY_READ_IN_DAYS;
+    public static final String REST_API_TOKEN_MAX_VALIDITY_WRITE_IN_DAYS;
+    public static final Boolean REST_API_TOKEN_GENERATOR_ENABLE;
+    public static final Boolean REST_API_WRITE_ACCESS_TOKEN_ENABLE;
+    public static final Set<String> DOMAIN;
+    public static final String REST_REPORT_FILENAME_MAPPING;
+    public static final String REST_JWKS_ISSUER_URL;
+    public static final String REST_JWKS_ENDPOINT_URL;
+    public static final Boolean REST_IS_JWKS_VALIDATION_ENABLED;
+    public static final String REST_SERVER_PATH_URL;
+
+    // UI Portlet Constants
```

Comment:
I was just wondering if UI constants are required anymore or it can be sanitized?

Reply:
I will recheck once which constants from this section are used by backend and which are not. The ones not used can be moved to the frontend project.

My initial thought was to have minimal changes the existing `sw360.properties` files. But yes, the best approach would be to separate the backend and frontend configs.